script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -1226,12 +1318,14 @@ She was happy because she had never experienced that trivial repetition until no
 [sha:ea2e895e4055a73dca245bfc870309efb81ff55f]{
 -- Page 124, Offset 29852.
 -- なにが悔しいかって―――あいつ自身、自分がどんなに馬鹿げた道を歩いてきたかを、これっぽっちも感じていない事だ。
-What frustrating convenience―――No matter what foolish path she walked down, she had never felt even that much.
+What's so incredibly frustrating―――is that she herself didn't even have an inkling of how absurd the path she'd been walking was.
```

Comment:
separating with the cjk dashes like this doesn't work with this new sentence structure. Just drop it